src/main/java/module-info.java

```diff
@@ -25,6 +25,9 @@
     opens com.movie_collection.config to com.google.guice, javafx.fxml;
     exports di to javafx.graphics;
     opens di to com.google.guice, javafx.fxml;
```

Comment:
yeah later we need to check and fix so its properly configured not just randomly :)